core/dbt/contracts/graph/parsed.py

```diff
@@ -626,6 +629,12 @@ class ParsedSourceDefinition(
     unrendered_config: Dict[str, Any] = field(default_factory=dict)
     relation_name: Optional[str] = None
     created_at: float = field(default_factory=lambda: time.time())
+    _event_status: Dict[str, Any] = field(default_factory=dict)
+
+    def __post_serialize__(self, dct):
+        if '_event_status' in dct:
+            del dct['_event_status']
+        return dct
```

Comment:
what is this and when is it called?

Reply:
@gshank can you speak to this?

Reply:
__post_serialize is a method that's called by the mashumaro serialization code. This removes the '_event_status' dictionary from the serialized form of the class (like manifest.json) because we should only need while running.